src/core/qfieldcloudconnection.cpp

```diff
@@ -246,7 +251,11 @@ void QFieldCloudConnection::login()
 
     mUsername = resp.value( QStringLiteral( "username" ) ).toString();
     emit usernameChanged();
-    mUserInformation  = CloudUserInformation( mUsername, resp.value( QStringLiteral( "email" ) ).toString() );
```

Comment:
This was added by @signedav , but if you think it is not critical, we can remove it. He still has debts to pay for the changelog :smiling_imp:

Reply:
You are a cruel human being 😉

All good.